vortexor/Readme.md

```diff
@@ -145,3 +145,80 @@ Operators can decide whether to adopt Vortexors without concerns about network
 protocol changes. Upgrading involves specifying the Vortexor's TPU address and
 verified packet receiver network address via CLI or Admin RPC. The transition is
 designed to be seamless for operators.
+
+# CLI Setup to pair a vortexor and a validator
```

Comment:
It would be far more clear in my opinion, to just share in the following way:

```md
To pair a validator and vortexor, you can add the following arguments to the validator cli command:

\```bash
    [INSERT ARGS HERE]
\```

and the following arguments to the vortexor command:

\```bash
	[INSERT ARGS HERE]
\```
```

Then explain what the flags/args do in detail. Right now with the disjoint discription it feels like we're reading code docs rather than a quick readme to tell me how to use it.

Reply:
done